### ISSUE TITLE
Postal Sending spam

### ISSUE THREAD

**Issue by raf-rodrigues** [postalhq/postal#1404]
I was blocked by Gmail from sending emails and I didn't understand as I have no complaints about spam or anything like that.

But, I received a spam sent by the server that I am having problems.

My SPF was marked as relachado I changed it to restricted.

But how to avoid this error?

[errorpostal](https://user-images.githubusercontent.com/39744108/125554459-a8884500-2b5f-400a-874c-f397a161dc10.png)

**Comment by willpower232** [comment]
If the message went through Postal you should be able to track it down in the UI to see where it came from and how it was sent.

**Comment by raf-rodrigues** [comment]
I am not able to see these messages through the panel.

Would there be any place that could see these messages? For I can't find and no location.

I previewed the sender and smtp log and I can't see where these messages are coming from;

**Comment by raf-rodrigues** [comment]
Solved the problem.

I discovered the problem and why I didn't see spam messages on the dashboard.

Spam messages were being sent via rabbitm that's why it wasn't displayed on the dashboard. the server was being used as a relay.

Changed passwords and database users, Rabbit. Panel. Reset the SMTP keys.

in addition to updating the configuration file.

Another thing is also increased security on the server itself.
